src/mon/MgrMonitor.cc

```diff
@@ -887,6 +887,11 @@ bool MgrMonitor::prepare_command(MonOpRequestRef op)
       r = -ENOENT;
       goto out;
     }
+    if (pending_map.enabled_module(module)) {
+      ss << "module " << module << " is already enabled";
+      r = -EINVAL;
```

Comment:
Same as in the disable case, if the module is already enabled then we must return 0 to be idempotent.

Reply:
fixed.